Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Old god decisions

## Changelog:
- Added **Request Elementium**, a decision of the Old God offmaps.
- Changed **Sacrifice to Old God**'s, an offmap decision's, insanity values.
- **Sacrifice to Old God**, an offmap decision, gives insanity for sacrificed character's class level.
- **Request Power from Old God**, an offmap decision, can give corruption.
- Added **Request General from Old God**, a decision of the Old God offmaps.
- Fixed an issue where the landless patricians (only family holding) couldn't use the Old God offmap decisions.

## Script changelog (ignore if you can't read CK2 script):
- Changed `calculate_PREV_offmap_old_gods_currency_insanity_effect` to use new values.
- Added `offmap_from_potential_trigger`.
- Added `is_old_god_offmap_tag_trigger`.
- Added class tooltips to is_1_2_physical_class_trigger, is_1_2_magic_class_trigger, etc.

# How to test:
Test the balance of insanity points, is it too easy to get them and spam armies and generals or is it too hard and the abilities are rarely used. Also test balance of Requests you can make and if some are not too costly or too cheap.

Just a note: to access the Old Gods offmaps we talk about you need to pick the main Old God in the Intrigue tab being an Old Gods' follower. A new tab with an eye should appear to the left of the minimap.

More notes, use these console commands to get offmap currency:
- `add_offmap_currency offmap_cthun 10000` to get 10000 C'thun's currency.
- `add_offmap_currency offmap_nzoth 10000` to get 10000 N'zoth's currency.
- `add_offmap_currency offmap_yoggsaron 10000` to get 10000 Yogg-Saron's currency.
